pkg/apis/ocibuilder/v1alpha1/types.go

```diff
@@ -81,6 +81,10 @@ type OCIBuilderSpec struct {
 	// Push contains specification to push images to registries
 	// +optional
 	Push []PushSpec `json:"push,omitempty" protobuf:"bytes,4,name=push"`
+	// Type of the build framework.
+	// Defaults to docker
+	// +optional
+	Daemon Daemon `json:"daemon,omitempty" protobuf:"bytes,5,opt,name=daemon"`
```

Comment:
Also is having `Daemon` as a separate type just for bool necessary? Usually, you would have a custom type and list of possible values.

Reply:
Also, the Daemon will always be false if the user doesn't provide value explicitly which leads to 
```

if !spec.Daemon {
   builder = "buidah"
}
```

always true.

Reply:
Set default to true